signer/tests/integration/transaction_signer.rs

```diff
@@ -170,3 +183,95 @@ async fn should_be_able_to_participate_in_signing_round() {
         .collect()
         .await;
 }
+
+/// Test that [`TxSignerEventLoop::get_signer_public_keys`] falls back to
```

Comment:
Should this be falling back to DKG as well if no rotate-keys tx?

Reply:
It doesn't need to. That function wants to return the currently known signing set on the stacks network. If there are no confirmed rotate-keys transaction on the canonical stacks blockchain then the Stacks network will think the signing set is the ones from our config.

Reply:
Ah okay, thx